ui/imports/utils/Constants.qml

```diff
@@ -899,9 +901,16 @@ QtObject {
     readonly property string networkRopsten: "Ropsten"
 
     readonly property string ethToken: "ETH"
-
-    readonly property int ethTokenWeiDecimals: 18
-    readonly property int ethTokenGWeiDecimals: 9
+    readonly property string bnbToken: "BNB"
+    readonly property string usdcToken: "USDC"
+    readonly property var rawDecimals: {
+        "ETH": 18,
+        "BNB": 18,
+    }
+    readonly property var gasTokenDecimals: {
```

Comment:
I am not sure why we need the concept of "gasTokenDecimals", in my opinion, it should not exist. It just brings confusion. I understand that it dates from the past, and should be completely gone after we remove `convertToOldRoute` on the Nim side, but that doesn't mean the new code should add more complexity.

All values coming from the backend should be in Wei format, actually the smallest denomination according to the token's decimals. On the UI side, it depends on the representation that we can convert it to the format we need.

So can we at last remove it from this PR (including related functions from the Util.qml) and use appropriate decimals where needed?

Reply:
this is supposed to represent what units you use for Gas display purposes. Eth uses Gwei (10ˆ-9), BNB uses the same, but other chains' native tokens might use another one (for example, for Bitcoin I see Sats or decimal BTC depending on the website)

I see we've got a mixture of stuff in decimal, raw and gas units across the app, ideally we would represent everything in raw units and only do the conversion at the display point. But that's not easily doable, some stuff comes from the Nim side in "gwei" already, so it would be a bigger rework to unify.

Does this make sense?

Reply:
I see what you're saying, but was thinking about using the raw value everywhere and converting it when needed to a necessary representation. But yes, removing `convertToOldRoute` on the Nim side and keeping it that way will be a new PR.